.env.sample

```diff
@@ -19,4 +19,7 @@ GOOGLE_ID='87822198681xxxxxxxxxxxxxxsercontent.com'
 GOOGLE_CALLBACK_URL='http://localhost:3000/login/callback'
 TWITTER_ID='ghgu'
 TWITTER_SECRET='ggug'
-TWITTER_CALLBACK_URL='ghgfh'
\ No newline at end of file
+TWITTER_CALLBACK_URL='ghgfh'
```

Comment:
Why are we re-adding this? This was merged yesterday.

Reply:
I removed it.